src/CenterLines.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+
+import { Row, Col } from "react-bootstrap";
+
+export function CenterLines(props) {
+  return (
+    <Row style={{ padding: "5vh" }}>
+      <Col xs={12} md={12} lg={12} style={{ textAlign: "center" }}>
```

Comment:
so just a quick note on bootstrap here, the col sizes cascade, so in this case you would only have to define `xs={12}` and then it would also be set for md, lg, etc.